src/www/vpn_openvpn_server.php

```diff
@@ -1523,6 +1523,17 @@
                     <tr>
                       <td colspan="2"><strong><?=gettext("Advanced configuration"); ?></strong></td>
                     </tr>
+                    <tr class="table table-striped opnsense_standard_table_form">
+                      <td style="width:22%"><a id="help_for_dynamic-ccd-lookup" href="#" class="showhelp"><i class="fa fa-info-circle"></i></a> <?=gettext("Use dynamic CCD Lookup"); ?></td>
+                      <td>
+                          <input name="dynamic-ccd-lookup" type="checkbox" value="1" <?=!empty($pconfig['dynamic-ccd-lookup']) ? "checked=\"checked\"" : "" ;?> />
```

Comment:
`'checked="checked"'` is better readable

Reply:
You mean, i should not copy & paste from above and improve all the ~ 15 cases where this looks like that? :)

I fixed that, but i am serious, i am just copying code and i am not really looking to improve a file which is a template, controller and inline-JS defintion is "stylistics". Its like it is, would be my way to deal with it :)